src/parsed.rs

```diff
@@ -232,3 +232,47 @@ impl<'a> ListVisitor<'a> for List {
         Ok(self)
     }
 }
+
+/// Serializes a structured field value into a string.
+pub trait SerializeValue {
```

Comment:
I'm not sure I understand [Move SerializeValue trait to parsed module to simplify feature checks](https://github.com/undef1nd/sfv/pull/156/commits/fe5f0037774f2433ee054eaef49c36d8f5d02fd1)

I would expect the SerializeValue trait to be in serializer.rs
Is this just so we can avoid a separate `#[cfg(feature = "parsed-types")]` check for for `pub use serializer::SerializeValue;` ?

Reply:
It avoids the separate feature check on the `pub use` and on the trait and trait impls, essentially confining almost all of the `parsed-types` code to `parsed.rs`.

Reply:
I don't mind the checks, in fact I think it makes it easier to see which things are behind the feature checks.
I would prefer to move them back, but that's up to you.

Reply:
Reverted.